include/lai/internal-ns.h

```diff
@@ -92,6 +101,12 @@ typedef struct lai_nsnode
             uint32_t pblk_addr;
             uint8_t pblk_len;
         };
+        struct { // LAI_NAMESPACE_OPREGION
+            uint8_t op_address_space;
+            uint64_t op_base;
+            uint64_t op_length;
+            struct lai_opregion_override op_override;
```

Comment:
Instead of this `struct lai_opregion_override`, we should store 1) a `const lai_opregion_override *` and 2) the userptr. `lai_opregion_override` should be adapted to take these data. The rationale is that `struct lai_opregion_override` is large (72 byte on x86_64) and this struct is stored for *each* `nsnode_t`.

After this change, users would just store a global `struct lai_opregion_override` (one for each driver) but could still pass a different userptr for each `nsnode_t`. It would also allow us to get rid of `->overridden` (a non-NULL pointer would always mean that the opregion is overridden).